.github/workflows/ci.yml

```diff
@@ -28,6 +28,9 @@ jobs:
       - name: Install
         run: yarn install
 
+      - name: Build
```

Comment:
naming it "Build" may be slightly confusing for our future selves, as it may suggest that building is a necessary step in our CI, while this is mainly smoke testing that webpack building step works, but we are not actually shipping the built file in the npm package we are releasing and so I'd name it "Build webpack bundle for the web" or "Smoke test webpack bundled module for the web" to make that more immediately clear.